include/mesh/distributed_mesh.h

```diff
@@ -643,6 +643,13 @@ class DistributedMesh : public UnstructuredMesh
    */
   bool _is_serial_on_proc_0;
 
+  /**
+   * A boolean remembering whether we've recently deleted top-level elements or
+   * not.  If so, we'll need to be extra careful when deleting "unused" nodes
```

Comment:
It's foolish that this seems to be changing as a standard.  Separating connected chunks of text by a level which scales with the strength of the connection is a good thing for readability.  Chapters get a page break and a heading.  Paragraphs get a blank line or at least an indentation.  Sentences get a punctuation and an extra space, and independent clauses only get punctuation.  The semantic connection between `not.  If` is weaker than the connection between `so, we'll`, and so the syntactical connection certainly should be weaker too.

Reply:
In LaTeX, the single-space-between-sentences option is called `\frenchspacing`, because the luminary of computer science and typesetting couldn't imagine that there might someday be *two* countries who thought that was a good idea.

Reply:
Haha I did intentionally not group this with the "requested changes" 😄 as I feel that having it this way *is* valid